pom.xml

```diff
@@ -29,7 +29,7 @@
 
   <groupId>org.everit.osgi.dev</groupId>
   <artifactId>eosgi-maven-plugin</artifactId>
-  <version>3.3.3</version>
+  <version>3.4.0-SNAPSHOT</version>
```

Comment:
Bump the version to 4.0.0-SNAPSHOT, this functionality will be released in that version.